docs/scaling/README.md

```diff
@@ -21,3 +21,11 @@ autoscaling.knative.dev/maxScale: "10"
 ```
 
 You can also use these annotations directly on `kpa` objects.
+
+**NOTE**: These annotations apply for the full lifetime of a `revision`.
+Even though a `revision` is not referenced by any `route`, a minimal pod count
+as specified by `autoscaling.knative.dev/minScale` is guaranteed. Non-routed
+`revisions` will be garbage collected after 24 hours(can be configured by
+`stale-revision-create-delay` in [config-gc.yaml](/config/config-gc.yaml)).
+If you want save resource for a non-routed `revision` before it is garbage
+collected, you should manually delete it.
```

Comment:
This is a bit of a nit so up to you on changing it, but I worry theres a more nuance to this comment about GC and its going to be hard to really explain it well in a short note e.g. GC only applies the non-routed revisions which have been routed at some point. This makes me thing it may be better to change the whole blurb starting with `non routed revisions` to `Keep in mind GC can also effect the lifecycle of revisions, for more information see [config-gc.yaml](/config/config-gc.yaml)`

Reply:
I agree. But `config-gc.yaml` doesn't provide detail information to me, e.g. as you mentioned, GC only applies the non-routed revisions which have been routed at some point. I'll keep as it is.

Reply:
Done